packages/react-sdk/src/components/BoardBar/ExportWhiteboardDialogDownloadPdf.tsx

```diff
@@ -114,14 +113,13 @@ function useGeneratePdf(
         setDownloadUrl(url);
       },
       error: (e) => {
+        // Make sure the URL is also revoked on error, so we don't leak
+        if (url) {
+          URL.revokeObjectURL(url);
+        }
         const logger = getLogger('ExportWhiteboardDialogDownloadPdf');
         logger.error('Error while generating the PDF', e);
-        onError?.(
-          t(
-            'boardBar.exportWhiteboardDialog.pdfError',
-            'Something went wrong while generating the PDF.',
-          ),
-        );
+        onError?.(e.message);
```

Comment:
moving this around as the error is generic and `t` was causing a re-render loop in production builds when used vite in the standalone project.